### DISCUSSION TITLE
is it possible to change corner radius for application style and plsama style?

### DISCUSSION THREAD

**Discussion by Tracenji** [paulmcauley/klassy#148] [Q&A] [answered]
i wasn't able to find any such settings in klassy-settings, is there any other way to change the radius settings on those two?

**Comment by paulmcauley** [comment, marked as answer]
It will be available for application style in the next 6.2 release. See 37b5101f2e1e308f10487b64397951938486fddd

Plasma style is a bit trickier programmatically as that involves editing SVGs, but you can always do that manually.
